transform/operations.go

```diff
@@ -347,12 +347,12 @@ func (c *stringToFloat64) init(args map[string]string) error {
 func (c *stringToFloat64) transform(raw interface{}) (interface{}, error) {
 	in, ok := raw.(string)
 	if !ok {
-		return nil, fmt.Errorf("stringToFloat64 only supports strings")
+		return nil, errors.New("stringToFloat64 only supports strings")
 	}
 
 	f, err := strconv.ParseFloat(in, 64)
 	if err != nil {
-		return nil, fmt.Errorf("error converting string to float64: %v", err)
+		return nil, fmt.Errorf("error converting string to float64: %T", err)
```

Comment:
You still need `%v` on this one because the `err` isn't a type.  The `%T` was for the first error printing, using `raw` as the argument.